README.md

```diff
@@ -209,6 +225,22 @@ Here's my setup for toggleterm, including an autocmd to automatically close a gi
 
 ```
 
+#### One instance per Kitty/Wezterm
+
+```lua
+    pipe_path = function()
+      -- If running in a terminal inside Neovim:
+      if vim.env.NVIM then return vim.env.NVIM end
+      -- If running in a Kitty terminal,
+      -- all tabs/windows/os-windows in the same instance of kitty will open in the first neovim instance
+      if vim.env.KITTY_PID then
```

Comment:
i don't really know enough about this area, what kind of issues could happen in this way?

Reply:
I don't know specifically what could go wrong, but attempting to start the server and allowing it to fail when a check could be made instead just doesn't make sense. Always better to avoid errors instead of catching them after the fact.